### PR TITLE
Build upstream Knative eventing operator 0.13.1

### PR DESCRIPTION
**Pull request by aliok**

### CODE REVIEW

**Comment by aliok** [comment]
/assign @droslean

cc @matzew

File: ci-operator/config/openshift-knative/eventing-operator/openshift-knative-eventing-operator-release-0.13.1.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    tag: golang-1.13
+images:
+- from: base
+  to: knative-eventing-operator
+promotion:
+  name: knative-v0.13.1
+  namespace: openshift
+resources:
+  '*':
+    limits: {}
```
Comment (droslean) [inline ci-operator/config/openshift-knative/eventing-operator/openshift-knative-eventing-operator-release-0.13.1.yaml]: no need to put empty values here.

**Comment by droslean** [comment]
Please use `make jobs` to generate your jobs

**Comment by aliok** [comment]
@droslean  thanks for the feedback. I did address your comments, but no idea what fails now. Could you have a look please?

```
2020-03-30 11:19:30,699:INFO:checking jobs ...
Traceback (most recent call last):
  File "./hack/ensure_job_cluster.py", line 134, in <module>
    main()
  File "./hack/ensure_job_cluster.py", line 128, in main
    ensure(args.job_dir, args.overwrite)
  File "./hack/ensure_job_cluster.py", line 102, in ensure
    .format(file_path, name_map))
Exception: those jobs in ./ci-operator/jobs/openshift-knative/eventing-operator/openshift-knative-eventing-operator-release-0.13.1-presubmits.yaml have to run on the cluster {'pull-ci-openshift-knative-eventing-operator-release-0.13.1-images': 'api.ci'}
```

File: ci-operator/jobs/openshift-knative/eventing-operator/openshift-knative-eventing-operator-release-0.13.1-presubmits.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
+presubmits:
+  openshift-knative/eventing-operator:
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - release-0.13.1
+    cluster: ci/api-build01-ci-devcluster-openshift-com:6443
```
Comment (droslean) [inline ci-operator/jobs/openshift-knative/eventing-operator/openshift-knative-eventing-operator-release-0.13.1-presubmits.yaml]: you have to remove that.

**Comment by droslean** [comment]
Please use `make jobs` to generate the jobs.

**Pull request merged**